doc/user/content/sql/create-connection.md

```diff
@@ -142,6 +142,8 @@ Field                       | Value            | Required | Description
 `PORT`                      | `int4`           | ✓        | Port for the connection.
 `USER`                      | `text`           | ✓        | Username for the connection.
 
+After executing the `CREATE CONNECTION my_tunnel FOR SSH TUNNEL ...` command, the public keys to be placed on the bastion server can be found by running `SELECT * FROM mz_ssh_tunnel_connections;`.
```

Comment:
```suggestion
#### Public key retrieval

Once a connection is created, you can find and retrieve the public keys to be used in the bastion server by running:

`SELECT * FROM mz_ssh_tunnel_connections;`

```

Reply:
We should also mention how to map `id` to the matching source, in case there are multiple SSH connections.